src/js/bootstrap-slider.js

```diff
@@ -1851,7 +1858,17 @@ const windowIsDefined = (typeof window === "object");
 			_toPercentage: function(value) {
 				return this.options.scale.toPercentage.apply(this, [value]);
 			},
+			_getTooltipPositionClassMap() {
+				const prefix = this.options.bsVersion === 'bs4' ? 'tooltip-' : '';
```

Comment:
could you wrap this conditional in parenthesis? makes the code a bit more readable

```js
const prefix = (this.options.bsVersion === 'bs4') ? 'tooltip-' : '';
```